sklearn/tests/test_common.py

```diff
@@ -43,12 +42,42 @@ def test_all_estimator_no_base_class():
         assert_false(name.lower().startswith('base'), msg=msg)
 
 
-def test_all_estimators():
+""" def test_all_estimators(): FIXME!!
```

Comment:
Do you need help resolving these FIXMEs?

Reply:
I just need a time turner. Should be able to work on this tomorrow?

Reply:
Oh wow this is not resolved.. hm....